src/plugins/inspector/public/views/requests/components/details/req_code_viewer.tsx

```diff
@@ -88,6 +106,18 @@ export const RequestCodeViewer = ({ indexPattern, json }: RequestCodeViewerProps
               {openInConsoleLabel}
             </EuiButtonEmpty>
           )}
+          {shouldShowsearchProfilerLink && (
+            <EuiButtonEmpty
+              size="xs"
+              flush="right"
+              iconType="search"
```

Comment:
nit: I think `visBarHorizontal` might look a bit cooler as a "low res" version of how search profiler kinda looks.